x/perpetual/keeper/begin_blocker.go

```diff
@@ -78,15 +78,24 @@ func (k Keeper) ComputeFundingRate(ctx sdk.Context, pool types.Pool) (math.Legac
 	}
 
 	fixedRate := k.GetParams(ctx).FixedFundingRate
+	minimumFundingRate := k.GetParams(ctx).MinimumFundingRate
```

Comment:
line 80 and 81 can be made to call only once, its fetching params twice from kv store